### PR TITLE
Added Tasks 1-6

### PR DESCRIPTION
**Pull request by RenataTs**

### CODE REVIEW

**Review by balakov-yevhen — APPROVED** [review]
Cool! :)

File: ITEA HW-5/Program.cs [hunk 1]
```diff
@@ -1,12 +1,163 @@
...
+        static void Swap(ref int a, ref int b)
+        {
+            int temp = a;
+            a = b;
+            b = temp;
+        }
+
+        static int Task1(ref int[,] array)
+        {
+            int min = array[0, 0];
```
Comment (SamoilenkoS) [inline ITEA HW-5/Program.cs]: Reuse index method

File: ITEA HW-5/Program.cs [hunk 2]
```diff
@@ -1,12 +1,163 @@
...
+                    }
+                }
+            }
+
+            return min;
+        }
+
+        static int Task2(ref int[,] array)
+        {
+            int max = array[0, 0];
```
Comment (SamoilenkoS) [inline ITEA HW-5/Program.cs]: Same here

**Review by SamoilenkoS — APPROVED** [review]
Fix it

File: ITEA HW-5/Program.cs (hunk 1 above)
Comment (RenataTs) [reply ITEA HW-5/Program.cs]: Сhanged it

File: ITEA HW-5/Program.cs (hunk 2 above)
Comment (RenataTs) [reply ITEA HW-5/Program.cs]: This is also changed

**Pull request merged**